app/V1Module/presenters/AssignmentsPresenter.php

```diff
@@ -178,6 +188,13 @@ public function actionUpdateDetail(string $id) {
       throw new InvalidArgumentException("No entry for localized texts given.");
     }
 
+    if ($this->getHttpRequest()->getHeader("content-type") === "application/json") {
```

Comment:
Why is this necessary here?

Reply:
Because the meaning of empty `disabledRuntimeEnvironments` is ambiguous when multipart/form-data is used.

Reply:
But this is quite ugly... Can you at least make some helper method here on BasePresenter or something like that?